src/component/Header/index.jsx

```diff
@@ -15,9 +39,11 @@ const Header = () => (
     <Navbar.Collapse>
       <Nav className="ml-auto">
         <Image className={styles['setting-logo']} src={settingLogo} rounded />
-        <Nav.Link href="#setting">Setting</Nav.Link>
-        <Image className={styles['login-logo']} src={loginLogo} rounded />
-        <Nav.Link href="#login">Login</Nav.Link>
+        <Nav.Link href="#setting">
+          <SettingModal />
+        </Nav.Link>
```

Comment:
It doesn't make sense to render a `Modal` inside a `Nav.Link`. You probably should remove the `Nav.Link` altogether.

Additionally, you probably should remove the gear image as well because it looks out of place right now being outside the Settings button.

![image](https://user-images.githubusercontent.com/12049480/120883678-675dd880-c593-11eb-8b76-b92978ed6c34.png)